src/main/java/edu/ucsb/cs156/dining/controllers/DiningCommonsController.java

```diff
@@ -0,0 +1,150 @@
+package edu.ucsb.cs156.dining.controllers;
+
+import edu.ucsb.cs156.dining.entities.DiningCommons;
+import edu.ucsb.cs156.dining.errors.EntityNotFoundException;
```

Comment:
Why is this here?  What end user functionality does it support?

There should only be one endpoint for the dining commons and it should just be a proxy for the UCSB Dining Commons API endpoint (i.e. pass parameters to the service, and return what the service returns.)